docs/admin/upgrade/upgrade-installation-with-operator.md

```diff
@@ -27,24 +20,26 @@ metadata:
   name: knative-serving
   namespace: knative-serving
 spec:
-  version: "0.22"
+  version: "0.23"
 EOF
+```
 
-## Verifying the upgrade
+## Verifying an upgrade by viewing pods
 
-To confirm that your Knative components have successfully upgraded, view the status of their pods in the relevant namespaces.
-All pods will restart during the upgrade and their age will reset.
-If you upgraded Knative Serving and Eventing, enter the following commands to get information about the pods for each namespace:
+You can confirm that your Knative components have upgraded successfully, by viewing the status of the pods for the components in the relevant namespace.
 
-```bash
-kubectl get pods --namespace knative-serving
-```
+!!! note
+    All pods will restart during the upgrade and their age will reset.
+
+### Knative Serving
```

Comment:
This is a situation where I think content tabs would be awesome! 

Github isn't letting me make the recommendation, but let me know if you'd want me to make this change, basically: 

=== "Knative Serving" 
...
=== "Knative Eventing"